Dockerfile

```
@@ -45,30 +45,13 @@ RUN apk add --no-cache ruby \
   "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}"
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Haskell build for: erd
+# Install erd-go (https://github.com/kaishuu0123/erd-go) as replacement for erd (https://github.com/BurntSushi/erd)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Alpine 3.16 is the last known working version for building erd (dynamic linking)
-FROM alpine:3.16 AS erd-builder
+FROM golang:alpine as erd-builder
+RUN apk add git \
```

```suggestion
RUN apk add --no-cache git \
```

The `--no-cache` ensures that `apk` retrieve the latest package index to download the latest `git` available for this Alpine baseline.
It does not persist the index though, which is what we want for a container image (otherwise you would want to use `--update` instead)